translations/mod-fqm-manager/en.json

```
@@ -308,5 +308,20 @@
   "entityType.drv_holdings_record_details.holdings_temporary_location": "Holdings temporary location",
   "entityType.drv_holdings_record_details.holdings_temporary_location_id": "Holdings temporary location ID",
   "entityType.drv_holdings_record_details.holdings_statistical_code_ids": "Holdings statistical code ID list",
-  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list"
+  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list",
+
+  "entityType.src_organization_type": "Organization types",
+  "entityType.src_organization_type.id": "ID",
+  "entityType.src_organization_type.organization_types_name": "Type",
+  "entityType.drv_organization_details" : "Organizations - vendor info",
+  "entityType.drv_organization_details.last_updated": "Org last updated",
+  "entityType.drv_organization_details.code": "Org Code",
+  "entityType.drv_organization_details.name": "Org Name",
+  "entityType.drv_organization_details.organization_status": "Org status",
+  "entityType.drv_organization_details.description": "Org Description",
```

Non-first words should not be capitalized
```suggestion
  "entityType.drv_organization_details.code": "Organization code",
  "entityType.drv_organization_details.name": "Organization name",
  "entityType.drv_organization_details.organization_status": "Organization status",
  "entityType.drv_organization_details.description": "Organization description",
```